src/main/java/org/openstreetmap/atlas/geography/Rectangle.java

```diff
@@ -350,10 +347,49 @@ public Rectangle expandHorizontally(final Distance distance)
      */
     public Rectangle expandVertically(final Distance distance)
     {
-        final Location newLowerLeft = this.lowerLeft.shiftAlongGreatCircle(Heading.SOUTH, distance);
-        final Location newUpperRight = this.upperRight.shiftAlongGreatCircle(Heading.NORTH,
-                distance);
-        return forCorners(newLowerLeft, newUpperRight);
+        final double degreesLatitudeToShift = distance.asMeters()
+                / Distance.DISTANCE_PER_DEGREE_LATITUDE.asMeters();
+        final long meterBuffer = 1;
+
+        final Location oldLowerLeft = this.lowerLeft;
+        Distance southShiftDistance = distance;
+        // If the lowerLeft is about to be shifted south of the South Pole, stop it!
+        if (oldLowerLeft.getLatitude().asDegrees() - degreesLatitudeToShift <= Latitude.MINIMUM
+                .asDegrees())
+        {
+            logger.error(
```

Comment:
I would make it a WARN since that has the potential to flood a lot when anyone runs it on a lot of rectangles!

Reply:
👍